frontend/workflows-lib/lib/components/workflow/WorkflowListFilterDrawer.tsx

```diff
@@ -198,6 +215,30 @@ function WorkflowListFilterDrawer({
               },
             }}
           />
+          <TextField
+            fullWidth
+            id="template"
+            label="Template"
+            variant="outlined"
+            value={template}
+            error={!!errors.template}
+            onChange={(e) => {
+              handleChangeTemplate(e.target.value);
+            }}
+            slotProps={{
+              input: {
+                endAdornment: creator ? (
+                  <IconButton
+                    onClick={() => {
+                      handleChangeTemplate("");
+                    }}
+                  >
+                    <ClearIcon sx={{ fontSize: 20 }} />
+                  </IconButton>
+                ) : undefined,
+              },
+            }}
+          />
           <Autocomplete
             multiple
             options={[
```

Comment:
Does WorkflowListFilterDrawer has any associated tests that need to be modified or updated given this new functionality?